src/V3Clock.cpp

```diff
@@ -308,13 +308,6 @@ class ClockVisitor final : public AstNVisitor {
         }
         VL_DO_DANGLING(nodep->unlinkFrBack()->deleteTree(), nodep);
     }
-    virtual void visit(AstAlwaysPostponed* nodep) override {
-        if (AstNode* stmtsp = nodep->bodysp()) {
-            stmtsp->unlinkFrBackWithNext();
-            nodep->addNextHere(stmtsp);
-        }
-        VL_DO_DANGLING(nodep->unlinkFrBack()->deleteTree(), nodep);
-    }
```

Comment:
AstAlwaysPostponed is an AstNodeProcedure so this visit is redundant as it does the same as the one above handling AstNodeProcedure. I thought I can remove that too but it needs to stay in case the bock was not inclined due to profilecfuncs. Not really important to the patch as of now.

Reply:
Ok, I will do this separately.